calculation/gmhazard_calc/gmhazard_calc/rupture/rupture.py

```diff
@@ -103,3 +109,70 @@ def rupture_id_ix_to_rupture_id(
 ):
     """Converts rupture id ix to rupture ids"""
     return ensemble.get_rupture_ids(rupture_id_ind)
+
+
+def get_fault_header_points(fault: nhm.NHMFault):
+    srf_points = []
+    srf_header: List[Dict[str, Union[int, float]]] = []
```

Comment:
`: List[Dict[str, Union[int, float]]]`
Is this necessary? I thought this was like a parameter for a few seconds :thinking:

Reply:
@claudio525 Nothing wrong with it. I was just checking in case he put it accidentally as I haven't seen we specify the type on a variable.
But just like you said, that makes sense, just like what TypeScript does.

Reply:
@SnowNooDLe Ahh my bad, I did think that was a parameter, I should have read your message properly instead of just skimming it, my bad. 
I agree we don't havent usually done this before, but I think its a good idea for complex variables, like this one.

Reply:
@claudio525 All goo, I am more of a person who'd love to leave more comments/details.

Reply:
Ah, I see I was thinking that if it's a parameter we probably don't want a mutable objects as a default arguments. 

If we are specifying that variable type, I think I'd prefer typed variables to be declared before untyped variables.